src/components/standalone/dashboard/TrafficSummaryChart.vue

```diff
@@ -0,0 +1,88 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import {
+  Chart as ChartJS,
+  Title,
+  Tooltip,
+  Legend,
+  BarElement,
+  CategoryScale,
+  LinearScale
+} from 'chart.js'
+import { Bar } from 'vue-chartjs'
+import { computed } from 'vue'
+import { byteFormat1024 } from '@nethserver/vue-tailwind-lib'
+import { useThemeStore } from '@/stores/theme'
+
+const themeStore = useThemeStore()
+
+const props = defineProps({
+  labels: {
+    type: Array,
+    required: true
+  },
+  datasets: {
+    type: Array,
+    required: true
+  }
+})
+
+const options: any = {
+  // horizontal bar chart
+  indexAxis: 'y',
+  scales: {
+    x: {
+      ticks: {
+        // format traffic on chart ticks
+        callback: function (value: number, index: number, ticks: any[]) {
+          return byteFormat1024(value)
+        },
+        // font color according to theme: gray-200 or gray-700
+        color: themeStore.isLight ? '#374151' : '#e5e7eb',
```

Comment:
Instead of hard coding colors, we could reference tailwind configuration directly from javascript.
It's just a suggestion, never used it :shrug:  
Ref: https://tailwindcss.com/docs/configuration#referencing-in-java-script

Reply:
It doesn't seem to work, anyway I'd stick with the actual implementation considering this:
![image](https://github.com/NethServer/nethsecurity-ui/assets/4612169/3c43a29e-6f57-4c19-b857-724013e7ab56)